routers/web/web.go

```diff
@@ -1331,6 +1331,10 @@ func registerRoutes(m *web.Route) {
 				m.Get("/artifacts/{artifact_name}", actions.ArtifactsDownloadView)
 				m.Post("/rerun", reqRepoActionsWriter, actions.Rerun)
 			})
+			m.Group("/workflows/{workflow_name}", func() {
+				m.Get("/badge.svg", actions.GetDefaultBranchWorkflowBadge)
+				m.Get("/{branch_name}/badge.svg", actions.GetWorkflowBadge)
+			})
```

Comment:
You need to respond correct Content-Type, I didn't see it in the handler.

Reply:
For the record, it's `image/svg+xml`.